### PR TITLE
feat: add HR packager codes

### PR DESCRIPTION
**Pull request by benbenben2**
### What
Add addresses for Croatian packager codes

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -3898,6 +3888,25 @@ HTML
...
+SIRET : $packager_codes{$canon_tagid}{siret} - <a href="$packager_codes{$canon_tagid}{section}">Source</a>
+</p>
+HTML
+						;
+				}
+
+				if ($packager_codes{$canon_tagid}{cc} eq 'hr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{naziv odobrenog objekta    (approved establishment)}<br>
+$packager_codes{$canon_tagid}{adresa objekta (address)} $packager_codes{$canon_tagid}{mjesto i poštanski broj (town/postal code)} ($packager_codes{$canon_tagid}{županija (county)})
```
Comment (alexgarel) [inline lib/ProductOpener/Display.pm:3904]: `Bareword "town" not allowed while "strict subs"`
might come from this:

```suggestion
$packager_codes{$canon_tagid}{'adresa objekta (address)'} $packager_codes{$canon_tagid}{'mjesto i poštanski broj (town/postal code)'} ($packager_codes{$canon_tagid}{'županija (county)'})
```

Comment (benbenben2) [reply lib/ProductOpener/Display.pm]: Thanks @alexgarel!

I updated the header to remove spaces, parenthesis, dots, etc.

File: lib/ProductOpener/Display.pm [hunk 2]
```diff
@@ -3898,6 +3888,25 @@ HTML
...
+SIRET : $packager_codes{$canon_tagid}{siret} - <a href="$packager_codes{$canon_tagid}{section}">Source</a>
+</p>
+HTML
+						;
+				}
+
+				if ($packager_codes{$canon_tagid}{cc} eq 'hr') {
+					$description .= <<HTML
+<p>$packager_codes{$canon_tagid}{naziv_odobrenog_objekta_approved_establishment}<br>
+$packager_codes{$canon_tagid}{adresa_objekta_address} $packager_codes{$canon_tagid}{mjesto_i_poštanski_broj_town_postal_code} ($packager_codes{$canon_tagid}{županija_county})
```
Comment (stephanegigandet) [inline lib/ProductOpener/Display.pm:3904]: The accents in key names are probably causing Fatal error while critiquing "lib/ProductOpener/Display.pm": Can't call method "location" on an undefined value at /opt/perl/local/lib/perl5/PPIx/QuoteLike/Token/Interpolation.pm line 47.

Comment (benbenben2) [reply lib/ProductOpener/Display.pm]: updated

**Comment by alexgarel** [comment]
@stephanegigandet I let you review !

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**